.gitbook.yaml

```diff
@@ -21,4 +21,4 @@ redirects:
    contributors/onboarding/staff-on-boarding-guide: /operations/workplace/people/working-at-mattermost/onboarding.md
    contributors/mattercon: /contributors/join-us/recruiting#working-and-meeting-across-the-globe.md
    operations/messaging-and-math/developer-relations: /operations/research-and-development/product/developer-relations.md
-   contributors/join-us/mattercon/mattermost-code-of-conduct: /contributors/mattercon/mattermost-code-of-conduct.md
+   contributors/join-us/mattercon/mattermost-code-of-conduct: ./contributors/mattercon/mattermost-code-of-conduct.md
```

Comment:
Why do we need the period here but not for the other redirects? @justinegeffen

Reply:
I've noticed that some redirects don't work and after reading the Gitbook docs I noticed that a period is used in the example, so I'm just testing a theory on this specific redirect. :)